src/plugins/global-settings/components/global-settings-color-picker/index.js

```diff
@@ -0,0 +1,236 @@
+							{ ...props }
+							id={ props.id || 'text-area' }
+							onChange={ event => props.onChange && props.onChange( event.target.value ) }
+						/>
+					</BaseControl>
+				</div>
+			</div>
+		</div>
+	</div>
+)
+
+// Component used to add a Delete Style button at the bottom of the COlorPicker.
+const DeleteButton = props => (
+	<div className="ugb-global-settings-color-picker__text-name components-color-picker__body">
+		<div className="components-color-picker__controls">
+			<Button
+				className="ugb-global-settings-color-picker__delete-button"
+				isLink
+				onClick={ props.onClick }
+			>
+				Delete Style
+			</Button>
+		</div>
+	</div>
+)
+
+const ColorPickers = ( { colors } ) => {
+	const [ selectedIndex, setSelectedIndex ] = useState( null )
+	const [ isPopoverOpen, setIsPopOverOpen ] = useState( false )
+	const [ colorButtonAnchor, setColorButtonAnchor ] = useState( null )
+
+	const onChangeColor = data => {
+		const { colors: updatedColors } = cloneDeep( select( 'core/block-editor' ).getSettings() )
+		const { colorVar: existingColorVar } = updatedColors[ selectedIndex ]
+		const colorVar = `--stk-global-color-${ md5( Math.floor( Math.random() * new Date().getTime() ) ).substr( 0, 5 ) }`
+
+		// Overwrite the selected color to a new color.
+		updatedColors[ selectedIndex ].color = existingColorVar ? `var(${ existingColorVar }, ${ data.hex })` : `var(${ colorVar }, ${ data.hex })`
+		updatedColors[ selectedIndex ].fallback = data.hex
+
+		// Add a fallback and colorVar if not exists.
+		updatedColors[ selectedIndex ].colorVar = existingColorVar || colorVar
+		updatedColors[ selectedIndex ].fallback = data.hex
+
+		dispatch( 'core/block-editor' ).updateSettings( {
+			colors: updatedColors,
+		} )
+
+		// Update the global style variable values
+		doAction( 'stackable.global-settings.global-styles', updatedColors )
+	}
+
+	const onChangeStyleName = value => {
+		const { colors: updatedColors } = cloneDeep( select( 'core/block-editor' ).getSettings() )
+
+		// Overwrite the selected style name and slug to a new style name and slug.
+		updatedColors[ selectedIndex ].name = value
+		updatedColors[ selectedIndex ].slug = value
+
+		dispatch( 'core/block-editor' ).updateSettings( {
+			colors: updatedColors,
+		} )
+	}
+
+	const onClickOutside = event => {
+		if ( event.target !== colorButtonAnchor ) {
+			setIsPopOverOpen( false )
+		}
+	}
+
+	const onColorDelete = () => {
+		const { colors: updatedColors } = cloneDeep( select( 'core/block-editor' ).getSettings() )
+
+		// Delete the specific color based on the selected index.
+		updatedColors.splice( selectedIndex, 1 )
+
+		dispatch( 'core/block-editor' ).updateSettings( {
+			colors: updatedColors,
+		} )
+
+		setIsPopOverOpen( false )
+	}
+
+	const AddIcon = props => (
+		<Button
+			{ ...props }
+			isDefault
+			className="ugb-global-settings-color-picker__add-icon"
+			label="Add New Color"
+			icon={
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 190 190">
```

Comment:
you should be able to use the built-in icons here `icon="plus"` or `plus-alt`

Reply:
I am using the icon list thin plus icon here. The ```plus``` icon has thick lines.